src/modules/navigator/mission.cpp

```diff
@@ -567,15 +599,19 @@ Mission::set_mission_items()
 		/* move to land wp as fixed wing */
 		if (_mission_item.nav_cmd == NAV_CMD_VTOL_LAND
```

Comment:
Doesn't this case overlap with "move to landing waypoint before descent if necessary" below?